### PR TITLE
dev-util/cutter: new package

### PR DESCRIPTION
**Pull request by otakuto**
A Qt and C++ GUI for radare2 reverse engineering framework

Package-Manager: Portage-2.3.16, Repoman-2.3.6

### CODE REVIEW

**Comment by monsieurp** [comment]
Hi. Please file a bug for this package.

**Comment by monsieurp** [comment]
@kensington, @a17r: care to review this PR, QT experts?

**Comment by kensington** [comment]
> @kensington, @a17r: care to review this PR, QT experts?

eqmake5 would normally be run in src_configure. I suspect that by overriding src_compile with it, the actual compilation ends up happening during src_install when make is called.

**Comment by otakuto** [comment]
I fixed it.

File: dev-util/cutter/cutter-1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+inherit qmake-utils
+
+DESCRIPTION="A Qt and C++ GUI for radare2 reverse engineering framework"
+HOMEPAGE="http://www.radare.org"
+SRC_URI="https://github.com/radareorg/cutter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
```
Comment (titanofold) [inline dev-util/cutter/cutter-1.0.ebuild]: Alphabetical, please.

**Comment by otakuto** [comment]
Alphabeticalized.

**Pull request closed without merging**